### ISSUE TITLE
New instances don't have unique component addresses

### ISSUE THREAD

**Issue by TGRHavoc** [gamax92/OCEmu#28]
I don't know if it's just me but, when I create a new instance of a computer, I get the same UUID for the components..

I'm creating the instances with the command line (`ocemu %APPDATA%/ocemu/1` and `ocemu %APPDATA%/ocemu/2`), when I run this I get the following output:

![New emulator](http://i.imgur.com/B1M59Xs.png)

The output is nothing unusual as it's creating a new directory etc. However, the addresses that is generated always seem to be the same.

For example, my first instance addresses are

``` lua
    --Default components available to the computer.
    components {

      {"gpu", "9031cf95-7a59-4ed2-af2c-dbb5834d0317", 0, 160, 50, 3},
      {"modem", "e4bd8be6-9971-4c53-a32b-dc6204fb8be8", 1, false},
      {"eeprom", "38484736-b88a-4bd4-972a-dc0f86a812ba", 9, "lua/bios.lua"},
      {"filesystem", "8cd40387-d6a2-4a55-8b29-dcbc07559a8b", 7, "loot/openos", "openos", true, 1},
      {"filesystem", "e05fbfd8-f5ba-48d6-bf27-dd698803215d", -1, "tmpfs", "tmpfs", false, 5},
      {"filesystem", "34eb7b28-14d3-4757-b326-dd1609b0a92e", 5, nil, nil, false, 4},
      {"internet", "88763779-32eb-45d8-a725-ddc38b5e3000", 2},
      {"computer", "dc02f2c9-5103-4459-9b24-de700c0bb8d2", -1},
      {"ocemu", "308dae1a-701c-42da-8f23-de1d8db83fa3", -1},
      {"screen_sdl2", "84196a6b-8e34-415b-8322-deca0e66c675", -1, 80, 25, 3},
      {"keyboard_sdl2", "d8a526bb-ad4c-40dc-b721-df7790134e46", -1}
    }
```

and the new instance

``` lua
     --Default components available to the computer.
    components {

      {"gpu", "9031cf95-7a59-4ed2-af2c-dbb5834d0317", 0, 160, 50, 3},
      {"modem", "e4bd8be6-9971-4c53-a32b-dc6204fb8be8", 1, false},
      {"eeprom", "38484736-b88a-4bd4-972a-dc0f86a812ba", 9, "lua/bios.lua"},
      {"filesystem", "8cd40387-d6a2-4a55-8b29-dcbc07559a8b", 7, "loot/openos", "openos", true, 1},
      {"filesystem", "e05fbfd8-f5ba-48d6-bf27-dd698803215d", -1, "tmpfs", "tmpfs", false, 5},
      {"filesystem", "34eb7b28-14d3-4757-b326-dd1609b0a92e", 5, nil, nil, false, 4},
      {"internet", "88763779-32eb-45d8-a725-ddc38b5e3000", 2},
      {"computer", "dc02f2c9-5103-4459-9b24-de700c0bb8d2", -1},
      {"ocemu", "308dae1a-701c-42da-8f23-de1d8db83fa3", -1},
      {"screen_sdl2", "84196a6b-8e34-415b-8322-deca0e66c675", -1, 80, 25, 3},
      {"keyboard_sdl2", "d8a526bb-ad4c-40dc-b721-df7790134e46", -1}
    }
```

As you can see, there's literally no difference in the component's addresses.

**Comment by gamax92** [comment]
Fixed this in 6f98f6cc501202f11d8388b39abf5b539fbcfb19, windows zip's have been updated.

**Comment by TGRHavoc** [comment]
@gamax92 Was literally going to send a PR with the same changes 😛
